### PR TITLE
Communicate test case item responsible for failure

### PR DESCRIPTION
**Pull request by spenced**
```
======================================================================
FAIL: test_result:6
5, {'requirements': 'G.8272/PRTC-A', 'parameters': {'time-error-limit/%': 100,  (tests.vse_sync_pp.analyzers.test_ts2phc.TestTimeErrorAnalyzer.test_result:6
5, {'requirements': 'G.8272/PRTC-A', 'parameters': {'time-error-limit/%': 100, )
Test vse_sync_pp.analyzers.ts2phc.TimeErrorAnalyzer analyzer test result and analysis
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/dspence/vse-sync-pp.git/tests/vse_sync_pp/analyzers/test_analyzer.py", line 187, in method
    self.assertEqual(analyzer.result, result, msg=msg)
AssertionError: False != True : (item 5 in `expect`)
```

### CODE REVIEW

**Comment by jnunyez** [comment]
LGTM
Something may help is to filter a selected set of unit tests to run (e.g., run only ppsdpll unit tests).

**Review by jnunyez — APPROVED** [review]

File: tests/vse_sync_pp/analyzers/test_analyzer.py [hunk 1]
```diff
@@ -171,9 +171,10 @@ def method(self):
     def make_test_result(constructor, fqname, expect):
         """Make a function testing analyzer test result and analysis"""
         # pylint: disable=too-many-arguments
-        @params(*expect)
-        def method(self, dct):
+        @params(*enumerate(expect))
+        def method(self, idx, dct):
             """Test analyzer test result and analysis"""
+            msg = f'(item {idx} in `expect`)'
```
Comment (crwr45) [inline tests/vse_sync_pp/analyzers/test_analyzer.py:177]: I suggest adding the `fqname` to `msg`. It's already available in the scope and puts all the important information in a single message:

```suggestion
msg = f'(item {idx} in `expect` of {fqname})'
```

**Review by crwr45 — CHANGES_REQUESTED** [review]
Basic approach looks good. Is the plan to apply the same pattern to the Parser tests once agreed here?

**Comment by spenced** [comment]
Abandoning. (People aren't suffering because this hasn't been merged yet.)

**Pull request closed without merging**